app/common/alerts.py

```diff
@@ -14,12 +13,20 @@ class AlertViaGovNotify:
 
     @staticmethod
     def send(email, reference):
-        notifications_client = NotificationsAPIClient(settings.NOTIFICATION_COMBINED_KEY)
-        notifications_client.send_email_notification(email_address=email,
-                                                     template_id=settings.NOTIFICATION_TEMPLATE_ID,
-                                                     personalisation=None,
-                                                     reference=reference)
-        logger.info('Sent secure message email notification', user_uuid=g.user.user_uuid)
+
+        notification = {
+            "emailAddress": email,
+            "reference": reference
+        }
+
+        url = urlparse.urljoin(settings.RM_NOTIFY_GATEWAY_URL, settings.NOTIFICATION_TEMPLATE_ID)
+
+        auth = (settings.SECURITY_USER_NAME, settings.SECURITY_USER_PASSWORD)
+
+        response = requests.post(url, auth=auth, timeout=settings.REQUESTS_POST_TIMEOUT, json=notification)
```

Comment:
We should probably do some check on the response we're getting from the notify service
i.e.
```
if response.status_code != 200:
    (Do error stuff here)
```

Reply:
Just want to clarify with someone after our chat that it's not meant to fail quietly and always return a 201. Any exception is swallowed in the AlertUser class which provides a switch for gov notify and merely logging.

Reply:
I definitely don't think that the error should just be swallowed with no other action. This email is sent after the message has been stored in the database so we need to:
- Alert ourselves that this email has not been sent (with logging/exceptions etc)
- Return a response from secure messaging which indicates that this has happened (different status code/error code maybe?)

This needs to happen on general exceptions in this chunk of code (ConnectionError etc) and when the notify service doesn't return a 200 status code

Reply:
I agree with this point. We should probably check the status code returned is healthy